lib/utils/open-url-prompt.js

```diff
@@ -0,0 +1,69 @@
+const readline = require('readline')
+const opener = require('opener')
+
+function print (npm, title, url) {
+  const json = npm.config.get('json')
+
+  const message = json ? JSON.stringify({ title, url }) : `${title}:\n${url}`
+
+  npm.output(message)
+}
+
+// Prompt to open URL in browser if possible
+const promptOpen = async (npm, url, title, prompt, emitter) => {
+  const browser = npm.config.get('browser')
+  const isInteractive = process.stdin.isTTY === true && process.stdout.isTTY === true
+
+  try {
+    if (!/^https?:$/.test(new URL(url).protocol)) {
```

Comment:
Do we really want to support `http:`?  Everywhere else we've stopped using it.

Reply:
Oh I see this is copied from the other file.  This is probably not the place to fix it.